docusaurus/docs/components/block-ui.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Block UI
+summary: Component for blocking sections of an interface
+---
+
+[![Version](https://img.shields.io/npm/v/@availity/block-ui.svg?style=for-the-badge)](https://www.npmjs.com/package/@availity/block-ui)
+
+### Installation
+
+npm
+
+```bash
+npm install @availity/block-ui
+```
+
+Yarn
+
+```bash
+yarn add @availity/block-ui
+```
+
+### Usage
+
+#### Example
+
+```jsx
+import React, { useState } from 'react';
+import BlockUi from '@availity/block-ui';
```

Comment:
should the example include importing the css as well?

Reply:
I have the component import the file itself now so the user doesn't have to. I need to look into whether this has some downsides

Reply:
hmm good question. maybe @TheSharpieOne could explain why he architected it that way initially?

Reply:
It was done this way before importing CSS in JS was a common thing. Not everyone had CSS importers configured in webpack or w/e they might have been using so it was left up to the user to import it via link tag or import statement.

Reply:
Availity apps will have `@availity/workflow` to make sure they don't break from this change. Do we think it will be a big enough problem for other users that I should have the user import the css themselves still?

Reply:
I don't think it's a problem anymore. We can pretty much expect users to be able to handle css imports.